sql/core/src/test/scala/org/apache/spark/sql/artifact/StubClassLoaderSuite.scala

```diff
@@ -101,7 +102,7 @@ class StubClassLoaderSuite extends SparkFunSuite {
 
     // Creating a new class loader will unpack the udf correctly.
     val cl2 = new ChildFirstURLClassLoader(
-      Array(udfNoAJar),
+      Array(new File(udfNoAJar).toURI.toURL),
```

Comment:
Please remove unrelated changes.

Reply:
There are no unrelated changes. This change is necessary otherwise test suite fails to initialize.

Reply:
Would you mind explaining why?

Reply:
The jar is removed, so the entire suite fails to initialize (see change on line 29).